modules/claims_api/app/controllers/claims_api/v2/veterans/intent_to_file_controller.rb

```diff
@@ -76,7 +84,10 @@ def intent_to_file_options(type)
         # BGS requires at least 1 of 'participant_claimant_id' or 'claimant_ssn'
         def handle_claimant_fields(options:, params:, target_veteran:)
           claimant_ssn = params[:claimantSsn]
-
+          if claimant_ssn.present?
+            claimant_ssn = claimant_ssn.gsub('-', '')
```

Comment:
I presume people don't use any other separators (like phone numbers sometimes have `.` instead of `-`)?

Reply:
`claimant_ssn.delete('^0-9')` feels more idiomatic to me to whitelist only digits, if we're worried about such things. I'm actually a little surprised rubocop didn't flag the non-regex gsub use with an alternative like `tr` or `delete`. Maybe we have that cop turned off.

Reply:
Updated to use `delete` in [PR](https://github.com/department-of-veterans-affairs/vets-api/pull/11975) for API-24720.